xwiki-platform-core/xwiki-platform-oldcore/pom.xml

```diff
@@ -624,6 +624,14 @@
   </dependencies>
   <build>
     <plugins>
+      <!-- Skipped for now as it really take very long time -->
```

Comment:
Do you know how long it takes or you stopped it before the end? :)

Reply:
No, actually I indeed stopped it before the end.